gradle/libs.versions.toml

```diff
@@ -1,14 +1,15 @@
 [versions]
 cache-ri-impl = "1.1.1"
-managed-caffeine = "2.9.3"
-managed-ehcache = "3.8.1"
-managed-hazelcast = "4.2.5"
-managed-infinispan = "12.1.12.Final"
+managed-caffeine = "3.1.1"
+managed-ehcache = "3.10.2"
+managed-hazelcast = "5.1.3"
+managed-infinispan = "13.0.11.Final"
 
 [libraries]
 # Managed
 managed-caffeine = { module = "com.github.ben-manes.caffeine:caffeine", version.ref = "managed-caffeine" }
 managed-hazelcast = { module = "com.hazelcast:hazelcast", version.ref = "managed-hazelcast" }
+managed-ehcache = { module = "org.ehcache:ehcache", version.ref = "managed-ehcache" }
 managed-ehcache-clustered = { module = "org.ehcache:ehcache-clustered", version.ref = "managed-ehcache" }
```

Comment:
I believe ehcache has changed how they package their libraries now, as we need ehcache as well as ehcache-clustered

Reply:
Yeah, it used to be `compile`, but now it's `runtime`